fetch.bs

```diff
@@ -4522,6 +4600,9 @@ method, when invoked, must run these steps:
 
  <li><p><a for="header list">Delete</a> <var>name</var> from
  <a for=Headers>header list</a>.
+
+ <li><p>If <a for=Headers>guard</a> is "<code>request-no-cors</code>", then
+ <a for=Headers>remove privileged no-cors request headers</a> from <a>context object</a>.
```

Comment:
I'm not sure how consistent Fetch is about it, but I've started using "the context object" elsewhere.

Reply:
I also need to change this so headers are only removed if an item was actually deleted